dashboard/app/views/levels/_admin.html.haml

```diff
@@ -36,7 +36,7 @@
         %li (Cannot edit)
 
     %br/
-    This level is in #{@level.script_levels.count} scripts:
+    This level is in #{@level.script_levels.to_a.count} scripts:
```

Comment:
You sure there isn't some rails caching going on? In `dashboard-console`, I'm seeing the DB get hit with `@level.script_levels` or `@levels.script_levels.to_a.count`, as long as it is the first usage.
```
irb(main):001:0> ActiveRecord::Base.logger = Logger.new(STDOUT)
=> #<Logger:0x007fec4bf5af98 @progname=nil, @level=0, @default_formatter=#<Logger::Formatter:0x007fec4bf5aed0 @datetime_format=nil>, @formatter=nil, @logdev=#<Logger::LogDevice:0x007fec4bf5ad68 @shift_size=nil, @shift_age=nil, @filename=nil, @dev=#<IO:<STDOUT>>, @mutex=#<Logger::LogDevice::LogDeviceMutex:0x007fec4bf5acc8 @mon_owner=nil, @mon_count=0, @mon_mutex=#<Mutex:0x007fec4bf5abd8>>>>
irb(main):002:0> ActiveRecord::Base.connection_pool.clear_reloadable_connections!
=> nil
irb(main):003:0> @level = Script.get_from_cache(1).script_levels.first.level
=> #<Blockly id: 456808996, game_id: 11, name: "blockly", created_at: "2015-10-16 22:36:26", updated_at: "2015-10-16 22:36:26", level_num: "u_1_1", ideal_level_source_id: nil, solution_level_source_id: nil, user_id: nil, properties: {}, type: "Blockly", md5: nil, published: false, notes: nil>
irb(main):004:0> @level.script_levels
D, [2016-09-14T13:19:07.062667 #4453] DEBUG -- :   ScriptLevel Load (0.6ms)  SELECT `script_levels`.* FROM `script_levels` INNER JOIN `levels_script_levels` ON `script_levels`.`id` = `levels_script_levels`.`script_level_id` WHERE `levels_script_levels`.`level_id` = 456808996
irb(main):005:0> @level.script_levels.to_a.count
=> 1
```
```
irb(main):001:0> ActiveRecord::Base.logger = Logger.new(STDOUT)
=> #<Logger:0x007fb3d65a0f28 @progname=nil, @level=0, @default_formatter=#<Logger::Formatter:0x007fb3d65a0f00 @datetime_format=nil>, @formatter=nil, @logdev=#<Logger::LogDevice:0x007fb3d65a0e88 @shift_size=nil, @shift_age=nil, @filename=nil, @dev=#<IO:<STDOUT>>, @mutex=#<Logger::LogDevice::LogDeviceMutex:0x007fb3d65a0e60 @mon_owner=nil, @mon_count=0, @mon_mutex=#<Mutex:0x007fb3d65a0de8>>>>
irb(main):002:0> ActiveRecord::Base.connection_pool.clear_reloadable_connections!
=> nil
irb(main):003:0> @level = Script.get_from_cache(1).script_levels.first.level
=> #<Blockly id: 456808996, game_id: 11, name: "blockly", created_at: "2015-10-16 22:36:26", updated_at: "2015-10-16 22:36:26", level_num: "u_1_1", ideal_level_source_id: nil, solution_level_source_id: nil, user_id: nil, properties: {}, type: "Blockly", md5: nil, published: false, notes: nil>
irb(main):004:0> @level.script_levels.to_a.count
D, [2016-09-14T13:16:06.793107 #4424] DEBUG -- :   ScriptLevel Load (0.5ms)  SELECT `script_levels`.* FROM `script_levels` INNER JOIN `levels_script_levels` ON `script_levels`.`id` = `levels_script_levels`.`script_level_id` WHERE `levels_script_levels`.`level_id` = 456808996
=> 1
irb(main):005:0> @level.script_levels
=> #<ActiveRecord::Associations::CollectionProxy [#<ScriptLevel id: 1, level_id: nil, script_id: 1, chapter: 1, created_at: "2015-10-16 22:36:26", updated_at: "2015-10-16 22:36:26", stage_id: 1, position: 1, assessment: nil, properties: nil, named_level: nil>]>
```
On another note, upon further reading, I think the `to_a.count` is actually subideal code. For example, per commentary [here](http://batsov.com/articles/2014/02/17/the-elements-of-style-in-ruby-number-13-length-vs-size-vs-count/), we should be using `size` rather than `count` on the array, noting that this might be a noop per commentary [here](http://stackoverflow.com/questions/4550770/count-size-length-too-many-choices-in-ruby).